unit2.js

```diff
@@ -0,0 +1,79 @@
+
```

Comment:
First off, CSS and styling followed guideline provided in the rubric fairly well. Functionality wise, it definitely requires re-visit and rearranging of some functions. Some codes are misplaced and have following problems: 
1. Functions are not provided with correct argument which creates issues in many functions (i.e. film objects). 
2. Please explore more into use of different event listeners such as "change" which will help you solve some of glitches. 
3. Along with previous feedback, also explore into use of "event" which will help you identify some of necessary info for the app. You can easily pick up things such as user input and option value by using event.target.
4. Pay attention to the flow of the app. It's important to understand the flow of the app in order to use functions in right places. 

Please spend some time before we start unit 3 to review and revisit this assessment. It definitely requires some rework on functions, and with enough time you should be able to do it. It is crucial to understand DOM manipulation, API requests, JS (I know you hate JS), and CSS as those are some of the most crucial skills as a developer. Please reach out to me or other instructors if you have any questions or need help.